x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/step_define_summary.tsx

```diff
@@ -94,6 +98,18 @@ export const StepDefineSummary: FC<Props> = ({
             >
               <span>{searchItems.dataView.getIndexPattern()}</span>
             </EuiFormRow>
+            {isDatePickerApplyEnabled && timeRangeMs && (
+              <EuiFormRow
```

Comment:
Can this time picker be made not to go full width rather than taking up the full form row?

<img width="1236" alt="image" src="https://user-images.githubusercontent.com/7405507/215555414-75cbd05f-25ad-4550-b454-4c1aed29feee.png">